css/properties.json

```diff
@@ -5801,7 +5801,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/list-style"
   },
   "list-style-image": {
-    "syntax": "<url> | none",
+    "syntax": "<> | none",
```

Comment:
Should be `<image>` instead of `<>`

Reply:
that's odd, not quite sure how I managed to remove that, fixed anyway.